Abcbadq/sjsu_shop
# SJSU-1012, Profile

STORY:
>Use the Udemy course to follow the creation of a profile for the users.
>
>criteria:
>
>-create button to access profile (only visible when signed-in)
>-user has to be able to change info (PW, address, payment method, delete account)"

What I did:
>-Added a dashboard, only accessible to a logged in user
>-when visiting the dashboard for the first time, the user is asked to add info
>-after that, he/she can edit the info, view history (other story), view own recipes(other story) or delete the account
>-all inputs are validated. address fields, CC expiration and CVV code only check for string and or lenght, not for being correct
>-also added a shopping cart to the dashboard, could be used by the person doing the cart-story, or can be deleted.

To Verify:
>-Make sure the dashboard only shows the buttons it should when logged in / logged out
>-make sure all the user input is validated, error messages displayed if input is missing
>-try to access pages like "/dashboard" when not logged in. can you access them without permission?
>-try updating the profile. also check mLab database. is it correct?


@minhphan156 @vinnysenthil @Abcbadq  please verify
